src/sources/forus-webshop/scss/schagen/includes/shame.scss

```diff
@@ -246,7 +246,7 @@
     &.block-markdown {
         a:not(.button) {
             text-decoration: underline;
-            color: #77b431;
+            color: #efefef;
```

Comment:
add ffffff to also
<img width="490" alt="Screenshot 2023-01-25 at 2 10 04 PM" src="https://user-images.githubusercontent.com/8695780/214559820-95388a17-8be1-4d5f-ba64-34fd429bf32b.png">